lib/wasi/src/runners/wasi.rs

```diff
@@ -42,8 +43,20 @@ impl crate::runners::Runner for WasiRunner {
         let atom_name = container.get_atom_name_for_command("wasi", command_name)?;
         let atom_bytes = container.get_atom(&container.get_package_name(), &atom_name)?;
 
-        let compiler = Cranelift::default();
-        let mut store = Store::new(compiler);
+        #[cfg(feature = "wasi_cranelift")]
+        let mut store = Store::new(wasmer::Cranelift::default());
+        #[cfg(feature = "wasi_llvm")]
+        let mut store = Store::new(wasmer::Llvm::default());
+        #[cfg(feature = "wasi_singlepass")]
+        let mut store = Store::new(wasmer::Singlepass::default());
+        #[cfg(not(any(
+            feature = "wasi_cranelift",
+            feature = "wasi_llvm",
+            feature = "wasi_singlepass"
+        )))]
+        let mut store = {
+            return Err(anyhow::anyhow!("wasmer-wasi needs one of wasi_cranelift or wasi_llvm or wasi_singlepass features enabled").into());
+        };
```

Comment:
This is not correct.

```suggestion
        let mut store =  Store::default()
```

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3483/commits/a483d047de82c89a295d3fbf7ce813dfdea1ed32